src/Symfony/Component/Translation/Command/TranslationPushCommand.php

```diff
@@ -94,13 +94,13 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
+        $provider = $this->providers->get($input->getArgument('provider'));
+
         if (!$this->enabledLocales) {
-            throw new InvalidArgumentException('You must define "framework.translator.enabled_locales" or "framework.translator.providers.%s.locales" config key in order to work with translation providers.');
+            throw new InvalidArgumentException(sprintf('You must define "framework.translator.enabled_locales" or "framework.translator.providers.%s.locales" config key in order to work with translation providers.', parse_url($provider, \PHP_URL_SCHEME)));
```

Comment:
the scheme doesn't always match the key, does it?

Reply:
For the 3 existing providers, yes it does, but you are right, it might not match for future providers.

Reply:
let's make it a convention and they should all match :)